### PR TITLE
Nlp improvements

### PR DESCRIPTION
**Pull request by bethbertozzi**
Added Repeat Functionality:
added a repeat story, intent, and action. also added to domain. 

Other Improvements:
minor tweak to days sick intent
added mood unhappy

### CODE REVIEW

File: deepspeech/results.txt [hunk 1]
```diff
@@ -1 +1 @@
-nine months old
+nine months old
```
Comment (camwr) [inline deepspeech/results.txt]: I think it's complaining about this file for some reason?

Comment (camwr) [reply deepspeech/results.txt]: I have no clue how github works lmao just trying to do my reviewer duty

**Review by camwr — APPROVED** [review]

File: deepspeech/results.txt (hunk 1 above)
Comment (bethbertozzi) [reply deepspeech/results.txt]: awe man im not sure what the dif is

Comment (bethbertozzi) [reply deepspeech/results.txt]: @camwr i think it is just a file generated by deepspeech so i deleted

**Pull request merged**